chapter2/tests/t20/terraform/dev/route53.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_route53_record" "records" {
```

Comment:
not semantic name
a better name would be "alb_a_record"

Reply:
> not semantic name a better name would be "alb_a_record"

@dmitry-mightydevops
Done